enterprise/frontend/src/metabase-enterprise/data_editing/tables/use-table-editing.tsx

```diff
@@ -0,0 +1,23 @@
+import { useCallback, useState } from "react";
```

Comment:
Did you consider adding data grid editing functionality right in the DataGrid component? It can be beneficial to prevent column options being recreated when edit state changes because currently column options depend on `editingCellId`. There is no issue in the current table editing view with that but I had problems with it in [feat(sdk): Revamp TableInteractive visualization component: update header styles, add text wrapping, and row indices](https://github.com/metabase/metabase/pull/54399#top) because click popover position was reset if we changed column options